test/application.spec.ts

```diff
@@ -48,6 +48,20 @@ test.group('Application', () => {
     })))
   })
 
+  test('resolve the namespace directory from rc file content', (assert) => {
+    const app = new Application(__dirname, new Ioc(), {
+      namespaces: {
+        models: 'App/Models',
+      },
+      autoloads: {
```

Comment:
Let's call this key `aliases`. The `autoloads` was the old name and kept for backward compatibility only